ocm/__main__.py

```diff
@@ -247,9 +249,16 @@ def main():
     add_parser = maincmd_parsers.add_parser(
         'append',
         aliases=('a',),
-        help='appends resources or sources to component-descriptor',
+        help='appends resources, sources, or component-references to component-descriptor',
+    )
+    add_parser.add_argument(
+        'type',
+        choices=(
+            'r', 'resource',
+            's', 'source',
+            'cref', 'component-reference',
```

Comment:
`c` is still free - preferably stick to single-letter short-forms for the sake of consistency.